recipes/openmesh/sitedir.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/src/Python/CMakeLists.txt b/src/Python/CMakeLists.txt
+index e622ab8..22fb44a 100644
+--- src/Python/CMakeLists.txt
++++ src/Python/CMakeLists.txt
+@@ -89,8 +89,12 @@ IF(OPENMESH_BUILD_PYTHON_BINDINGS)
+ 
+ 						FILE(GLOB SOURCES *.cc *hh)
+ 						ADD_LIBRARY(openmesh SHARED ${SOURCES})
+-
+-						install(TARGETS openmesh DESTINATION ${ACG_PROJECT_LIBDIR}/python )
++                        if (WIN32)
```

Comment:
I don't think we need this patch as we can probably just move the files ourselves - no?